src/common/settings.js

```diff
@@ -283,6 +283,13 @@ const options_list = {
         // callbacks will be added after the functions are defined
     },
 
+    "astral-control-hp-bar": {
+        "title": "Control and sync Astral hp resource bars",
+        "description": "Beyond20 will control hp and temp hp resource bars and sync hp, it will use the first bar, the default one for hp, and the second one for temp hp.",
+        "type": "bool",
+        "default": true
+    },
+
```

Comment:
Why is this necessary? There's already the `update-hp` setting, so why not use that directly? IS there a reason to have two settings about updating the hp ? Or is this just for setting the bars, in which case is it necessary (i.e: can the user define the bars to match hp, and if they're not, then don't do anything, or if hp update is enabled, then always sync the hp bars too)?

Reply:
Added it at first to check some things out that they didn't break. Wanted to make it such that you could allow beyond20 to manage the resource bars or not and configure them manually in astral to reflect the updated stats. But sadly resource bars don't update their values after updating the hp stats through a post request, so we need to force update the bars too. I guess at this point it is vestigial from the initial purpose it had.